website/project/metadata/osf-standard-2.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
> will the bump in version number cause issues for older registration data?

Nope, because the old versions will still exist and existing registrations will be tied to them.

> If we alter the structure of the schema file defining questions (such as Eric's schema changes), will we have to alter any pages to deal with [..]

Yes, as the schema is effectively passed intact to the client. That said, the changes from iterating over a dict to iterating over a list shouldn't be huge - and they should be only in a few places.

In particular, those things are already stored as lists in `registrationUtils.js`, so only the serialization/deserialization logic will need to change.